deploy/webhook.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
```

Comment:
Note that when this is added to ACM, you'll want to use OpenShift's mechanism of generating certificates instead of Cert Manager:
https://docs.openshift.com/container-platform/4.13/security/certificates/service-serving-certificate.html#add-service-certificate_service-serving-certificate
https://docs.openshift.com/container-platform/4.13/security/certificates/service-serving-certificate.html#add-service-certificate-validating-webhook_service-serving-certificate

No action is needed in this PR though.